### PR TITLE
take care of deprecation messages

### PR DESCRIPTION
**Pull request by pgwillia**

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -28,6 +28,6 @@ class Application < Rails::Application
     # config.i18n.default_locale = :de
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
-    #config.active_record.raise_in_transactional_callbacks = true
+    config.active_record.raise_in_transactional_callbacks = true
```
Comment (mbarnett) [inline config/application.rb]: Have we actually tested that nothing breaks when we opt in to this new behaviour: http://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#error-handling-in-transaction-callbacks ?

I'm a bit reluctant to merge this and change the semantics of after_commit right before launch.

Comment (pgwillia) [inline config/application.rb]: I have no problem with waiting on this.  Just wanted to get it off my VM before I forgot or wiped it clean.

Comment (mbarnett) [inline config/application.rb]: Makes sense to me. Let's sit on this until after we're in prod.

**Comment by pgwillia** [comment]
Minor annoyances.  Will create a new PR when config.active_record.raise_in_transactional_callbacks is thought through.

**Pull request closed without merging**